paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentSheetLoader.kt

```diff
@@ -118,27 +118,50 @@ internal class DefaultPaymentSheetLoader @Inject constructor(
         val isLinkAvailable = stripeIntent.paymentMethodTypes.contains(Link.code) &&
             stripeIntent.linkFundingSources.intersect(supportedFundingSources).isNotEmpty()
 
+        val savedSelection = async {
+            prefsRepository.getSavedSelection(
+                isGooglePayAvailable = isGooglePayReady,
+                isLinkAvailable = isLinkAvailable,
+            )
+        }
+
         val paymentMethods = async {
             if (customerConfig != null) {
                 retrieveCustomerPaymentMethods(
-                    stripeIntent,
-                    config,
-                    customerConfig
+                    stripeIntent = stripeIntent,
+                    config = config,
+                    customerConfig = customerConfig,
                 )
             } else {
                 emptyList()
             }
         }
 
-        val savedSelection = async {
-            retrieveSavedPaymentSelection(
-                prefsRepository,
-                isGooglePayReady,
-                isLinkAvailable,
-                paymentMethods.await()
+        val sortedPaymentMethods = async {
```

Comment:
nit: can we name this `sortedSavedPaymentMethods` or just `savedPaymentMethods`

Reply:
How is this list being sorted, can we give the sort method a more descriptive name?

Reply:
Yep, will rename and add a test.